src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -71,10 +70,20 @@ public enum Operator {
        */
       MAP_EQUALS,
       /**
-       * This represents eq operation for array element or automic value operation against
+       * This represents eq to be run against map type index columns like array_size, sub_doc_equals
+       * and array_equals.
+       */
```

Comment:
The docs here is the same as `MAP_EQUALS`. Should it be changed to no eq?

Reply:
thanks